Documentation/toc.yml

```
@@ -81,7 +81,7 @@
     href: TODO.md
     items:
     - name: Performance
-      href: TODO.md
+      href: Performance/GettingStarted.md
```

The docs build is failing due to an invalid link.

```suggestion
      href: Performance/PerfGettingStarted.md
```